doc/userManual.md

```diff
@@ -195,16 +195,17 @@ The error explanation comes from the error LabVIEW internal database.
 
 ## Server Configuration Cluster
 
-In the following table you can find the parameters of the `ServerConfiguration` cluster:
+This cluster is a `typedef` named **configServerTypeDef&#46;ctl** and is called by the **configServer&#46;vi**.
+In the following table you can find the parameters to configure the server:
 
 | Parameter   | Description         |
 |:---------------:|:-------------------:|
 | cmdServerPort | TCP port used by native LabVIEW TCP functions in TcpServer Command |
 | telServerPort | TCP port used by native LabVIEW TCP functions in TcpServer Telemetry|
-| serverTimeout (ms) | Time in ms that the `TCP Wait on Listener` function waits for a TCP connection |
+| serverTimeout (ms) | Time in ms that the `TCP Wait on Listener` function waits for a TCP connection. This will be used in stopServer occurrence timeout and in the runServer Event Structure timeout, too. |
```

Comment:
Time in ms that ... --> Timeout in ms that ...

In addition, if it is not complex, consider to rename `serverTimeout (ms)` to `timeoutServerInMs`. This is for the consistency. Ignore this suggestion if it is complex or difficult.

Reply:
I made this changes. Thanks.